lib/src/turn_page_view.dart

```diff
@@ -63,7 +71,10 @@ class _TurnPageViewState extends State<TurnPageView>
       thresholdValue: widget.controller.thresholdValue,
       duration: widget.controller.duration,
     );
+    generatePages();
+  }
 
+  generatePages() {
```

Comment:
[should]
In our practice for this package, we opt to explicitly declare 'void' for better readability, even though it might seem a bit verbose. I would greatly appreciate if you could adhere to this convention by not omitting 'void' in your function declarations.
```suggestion
  void generatePages() {
```